src/opcodes/opcodes.cairo

```diff
@@ -486,4 +487,9 @@ pub mod Opcode {
     fn opcode_1negate(ref engine: Engine) {
         engine.dstack.push_int(-1);
     }
+
+    fn opcode_pick(ref engine: Engine) {
+        let a = engine.dstack.pop_int();
+        engine.dstack.push_int(a);
```

Comment:
This isn't fully accurate to how opcode_pick should work. I'd recommend reading this: https://github.com/btcsuite/btcd/blob/b161cd6a199b4e35acec66afc5aad221f05fe1e3/txscript/opcode.go#L1328

Lmk if you have questions